configs/gtp-base.cfg

```diff
@@ -17,12 +17,20 @@ allowResignation = false
 resignConsecTurns = 3
 resignThreshold = -0.90
 
-# The rest is copied from gtp_example.cfg; we need to copy instead of @include because
+# The below parameters are copied from gtp_example.cfg; we need to copy instead of @include because
 # it's currently impossible to override "rules = tromp-taylor" once it's set in an included file.
 lagBuffer = 1.0
 maxTimePondering = 60
 ponderingEnabled = false
-searchFactorAfterOnePass = 0.50
-searchFactorAfterTwoPass = 0.25
-searchFactorWhenWinning = 0.40
-searchFactorWhenWinningThreshold = 0.95
\ No newline at end of file
+# These parameters are part of gtp_example.cfg to weaken the bot for
+# friendliness, but we're commenting them out since friendliness tends to help
+# our adversary.
+# searchFactorAfterOnePass = 0.50
+# searchFactorAfterTwoPass = 0.25
+# searchFactorWhenWinning = 0.40
+# searchFactorWhenWinningThreshold = 0.95
+
+# These parameters are copied from match_example.cfg to make behavior
+# similar to behavior when running `match`.
+chosenMoveTemperatureEarly = 0.60
```

Comment:
Unclear if we should resolve disagreement in favor of match values or GTP -- but am OK going with match defaults for now pending response from lightvector.

Reply:
I'll remove this since  lightvector suggests that GTP values should be "default" values